README.md

```diff
@@ -23,10 +23,48 @@ npm install -g @creuna/cli
 
 ### Usage
 
+Print help:
+
 ```
 creuna
 ```
 
+Run commands:
+
+```
+creuna <command>
+```
+
+### Commands
+
+#### new \<relative-path>
+
+Creates a new React app in the **current working directory**. If a `path` is provided, files will be written to this path, **relative** to the current working directory
+
+#### lib
```

Comment:
this is a lot of markdown levels, are you doing typography with them?

Reply:
I an 😃